src/plugins/visualizations/public/legacy/build_pipeline.ts

```diff
@@ -80,7 +80,7 @@ const vislibCharts: string[] = [
   'line',
 ];
 
-const getSchemas = (
+export const getSchemas = (
```

Comment:
If we are exporting this publicly now, I wonder if we should name it something more specific (`getVisSchemas`)?